sdk/src/index.js

```
@@ -1,269 +1,777 @@
+      case 'input':
+        return { output: node.data?.value || '' };
+        
+      case 'output':
+        return { output: inputs.input || Object.values(inputs)[0] };
+        
+      case 'static-text':
+        return { output: node.data?.text || node.data?.value || '' };
+        
+      case 'combine-text':
+        const input1 = inputs.input1 || inputs.text1 || '';
+        const input2 = inputs.input2 || inputs.text2 || '';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)